extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js

```diff
@@ -124,19 +125,7 @@ require([
 		});
 
 		doc.addEventListener('bab.not_blocking', function () {
-			if (a9 && a9.waitForResponseCallbacks && adContext.get('bidders.a9Video')) {
-				a9.waitForResponseCallbacks(
-					function onSuccess() {
-						bidParams = a9.getSlotParams(featuredVideoSlotName);
-						setupPlayer();
-					},
-					function onTimeout() {
-						bidParams = {};
-						setupPlayer();
-					},
-					responseTimeout
-				);
-			} else if (bidders && bidders.addResponseListener && bidders.isEnabled()) {
+			if (bidders && bidders.addResponseListener && bidders.isEnabled()) {
```

Comment:
Correct me if I'm wrong, but it seems that `bidders.addResponseListener` isn't exposed 🙈

Reply:
Also I believe we don't need to expose addEventListener, but reuse logic from `waitForResponse` because it already has timeout-related logic

Reply:
Uf, it is: https://github.com/Wikia/app/pull/16526/files?w=1#diff-b35844bff9604cb4e285b9b3993e9e93R52

Reply:
Yes, it is. But this part need some refactor and changes in ad-engine. Fixing.